server_apps/data_transfer/PUBMED/loadNewPubs.sql

```diff
@@ -21,8 +21,8 @@ create temp table tmp_pubs (
   status text,
   pubType text);
 
-\copy tmp_pubs from '<!--|TARGETROOT|-->/server_apps/data_transfer/PUBMED/parsePubs.log';
-\copy (select zdb_id from tmp_pubs,publication where accession_no = pmid) to '<!--|TARGETROOT|-->/server_apps/data_transfer/PUBMED/pubAlreadyinZFIN.txt' delimiter '|';
+\copy tmp_pubs from '/Users/cmpich/projects/zfin/schlapp/server_apps/data_transfer/PUBMED/parsePubs.log';
```

Comment:
Looks like you lost ''<!--|TARGETROOT|-->'.  Could we use a relative path?

Reply:
Good catch. I think you can make the path of the file to read relative, not sure if that works for the output file write.